common/app/views/fragments/headTonal.scala.html

```diff
@@ -85,10 +84,7 @@ <h1 class="content__headline js-score" itemprop="headline">
                     </div>
                 }
 
-                @if(
-                    (showBadge && !item.commercial.isAdvertisementFeature) ||
-                        (staticBadgesSwitch.isSwitchedOn && !item.tags.isVideo && !item.tags.isAudio)
-                ) {
+                @if(showBadge && !item.commercial.isAdvertisementFeature) {
```

Comment:
Do we no longer care about whether there are video/audio tags on the page before we show the badge?